ReactWindows/ReactNative.Shared/Modules/Network/NetworkingModule.cs

```diff
@@ -392,20 +392,22 @@ private void OnResponseReceived(int requestId, HttpResponseMessage response)
         {
             var headerData = new JObject();
 #if WINDOWS_UWP
-            TranslateHeaders(headerData, response.Headers);
+            var responseHeaders = response.Headers;
 #else
-            IDictionary<string, string> headers = response.Headers.ToDictionary((kvp) => kvp.Key, (kvp) => kvp.Value.ToString());
-            TranslateHeaders(headerData, headers);
+            var responseHeaders = response.Headers.Select(pair => 
+                new KeyValuePair<string, string>(pair.Key, string.Join(", ", pair.Value)));
```

Comment:
FYI, just double checked, and the original code that calls "ToString()" will return "String[]", which is not expected behavior.